frontend/src/metabase/parameters/components/ParameterWidget/ParameterWidget.tsx

```diff
@@ -18,23 +19,25 @@ import {
 type ParameterWidgetProps = PropsWithChildren<
   {
     parameter: UiParameter;
-  } & Partial<{
-    setValue: (value: any) => void;
-    question?: Question;
-    dashboard?: Dashboard | null;
+  } & Partial<
+    {
+      setValue: (value: any) => void;
+      question: Question;
+      dashboard: Dashboard | null;
 
-    editingParameter: Parameter | null | undefined;
-    commitImmediately: boolean;
-    parameters: UiParameter[];
-    setParameterValueToDefault?: (parameterId: ParameterId) => void;
-    enableParameterRequiredBehavior?: boolean;
-    isSortable?: boolean;
-    isEditing?: boolean;
-    className?: string;
-    isFullscreen?: boolean;
-    setEditingParameter?: (parameterId: ParameterId | null) => void;
-    dragHandle: ReactNode;
-  }>
+      editingParameter: Parameter | null;
+      commitImmediately: boolean;
+      parameters: UiParameter[];
+      setParameterValueToDefault: (parameterId: ParameterId) => void;
+      enableParameterRequiredBehavior: boolean;
+      isSortable: boolean;
+      isEditing: boolean;
+      className: string;
+      isFullscreen: boolean;
+      setEditingParameter: (parameterId: ParameterId | null) => void;
+      dragHandle: ReactNode;
+    } & Pick<DashboardFullscreenControls, "isFullscreen">
+  >
 >;
```

Comment:
I removed `?` optional modifier from fields, because the whole object is wrapped inside `Partial`